gradle/libs.versions.toml

```
@@ -36,6 +36,7 @@ airbnb-showkase-processor = { module = "com.airbnb.android:showkase-processor",
 
 androidx-activity-compose = { group = "androidx.activity", name = "activity-compose", version.ref = "androidx-activity" }
 # https://androidx.tech/artifacts/appcompat/appcompat-resources/
+androidx-appCompat-appCompat = { module = "androidx.appcompat:appcompat", version.ref = "androidx-appCompat" }
```

Let's use the root name

```suggestion
androidx-appCompat = { module = "androidx.appcompat:appcompat", version.ref = "androidx-appCompat" }
```